src/Vision/ImageAnnotations.php

```diff
@@ -71,12 +75,15 @@ public function isIdentityDocument(): bool
 
     public function isFrenchNationalIdentityCard(): bool
     {
-        return $this->isIdentityDocument() && \in_array(self::FRENCH_IDENTITY_CARD_LABEL, $this->labels, true);
+        return $this->isIdentityDocument() && (
+            \count(\array_intersect(self::FRENCH_IDENTITY_CARD_LABELS, $this->labels)) > 0 ||
+            \in_array(self::FRENCH_LANGUAGE, $this->webEntities, true)
```

Comment:
on some test, my `labels` only contains `[document]` which is not helpful. However my idendity card was a good one, but Google seems to not parse correctly. So If it is a n identity document, this check allows now to pass if it find any of the french authorized labels or if it find the `French language` webEntity; What do you think ?

Reply:
label `French language` is not mandatory, no

Reply:
reverted